rel-eng/push-packages-to-obs.sh

```diff
@@ -290,8 +290,7 @@ while read PKG_NAME; do
           tar cf $CHART_TAR -C ${SRPM_PKG_DIR}/tar .
           rm -rf ${SRPM_PKG_DIR}/tar
       fi
```

Comment:
We need to add the Uyuni / SUMA semantic version as a build tag here too

Reply:
And here